File: app/src/main/java/com/nelyan/ui/PubilerActivity.java
package com.nelyan.ui;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Context;
import android.content.Intent;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;
import android.widget.LinearLayout;

import com.nelyan.R;

public class PubilerActivity extends AppCompatActivity {
Context mContext;
ImageView ivBack,ivLogo,ivRadio1,ivOn,ivRadio2,ivRadio3,ivRadio4;
LinearLayout ll_1,ll_2,ll_3,ll_4,ll_5;
Button btnSubmit;
int state =0;
String type="1";
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_pubiler);
        mContext = this;
        ivBack = findViewById(R.id.ivBack);
        ll_1 = findViewById(R.id.ll_1);
        ll_2 = findViewById(R.id.ll_2);
        ll_3 = findViewById(R.id.ll_3);
        ll_4 = findViewById(R.id.ll_4);
        ll_5 = findViewById(R.id.ll_5);
        ivOn = findViewById(R.id.ivOn);
        ivRadio1 = findViewById(R.id.ivRadio1);
        ivRadio2 = findViewById(R.id.ivRadio2);
        ivRadio3 = findViewById(R.id.ivRadio3);
        ivRadio4 = findViewById(R.id.ivRadio4);
        ll_1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                type = "1";

                ivOn.setImageResource(R.drawable.radio_outline);
                ivRadio1.setImageResource(R.drawable.radio_fill);
                ivRadio2.setImageResource(R.drawable.radio_outline);
                ivRadio3.setImageResource(R.drawable.radio_outline);
                ivRadio4.setImageResource(R.drawable.radio_outline);
            }
        });
        ll_2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                type = "2";
                ivOn.setImageResource(R.drawable.radio_fill);
                ivRadio1.setImageResource(R.drawable.radio_outline);
                ivRadio2.setImageResource(R.drawable.radio_outline);
                ivRadio3.setImageResource(R.drawable.radio_outline);
                ivRadio4.setImageResource(R.drawable.radio_outline);
            }
        });
        ll_3.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                type = "3";
                ivOn.setImageResource(R.drawable.radio_outline);
                ivRadio1.setImageResource(R.drawable.radio_outline);
                ivRadio2.setImageResource(R.drawable.radio_fill);
                ivRadio3.setImageResource(R.drawable.radio_outline);
                ivRadio4.setImageResource(R.drawable.radio_outline);
            }
        });
        ll_4.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                type = "4";
                ivOn.setImageResource(R.drawable.radio_outline);
                ivRadio1.setImageResource(R.drawable.radio_outline);
                ivRadio2.setImageResource(R.drawable.radio_outline);
                ivRadio3.setImageResource(R.drawable.radio_fill);
                ivRadio4.setImageResource(R.drawable.radio_outline);
            }
        });
        ll_5.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                type = "5";
                ivOn.setImageResource(R.drawable.radio_outline);
                ivRadio1.setImageResource(R.drawable.radio_outline);
                ivRadio2.setImageResource(R.drawable.radio_outline);
                ivRadio3.setImageResource(R.drawable.radio_outline);
                ivRadio4.setImageResource(R.drawable.radio_fill);
            }
        });


        ivBack.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                finish();
            }
        });
        ivLogo = findViewById(R.id.ivLogo);
        ivLogo.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent i = new Intent(PubilerActivity.this, HomeActivity.class);
                startActivity(i);
            }
        });

        btnSubmit = findViewById(R.id.btnSubmit);
        btnSubmit.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                if (type.equals("1")) {
                    Intent i = new Intent(PubilerActivity.this, Activity3Activity.class);
                    startActivity(i);
                } else if (type.equals("2")) {
                    Intent i = new Intent(PubilerActivity.this, NurserieActivity.class);
                    startActivity(i);
                } else if (type.equals("3")) {
                    Intent i = new Intent(PubilerActivity.this, MaternalAssistantActivity.class);
                    startActivity(i);
                } else if (type.equals("4")) {
                    Intent i = new Intent(PubilerActivity.this, BabySitterActivity.class);
                    startActivity(i);
                } else if (type.equals("5")) {
                    Intent i = new Intent(PubilerActivity.this, TraderActivity.class);
                    startActivity(i);
                }
            }
        });
    }
}

File: app/src/main/java/com/nelyan/ui/PubilerActivity.kt
package com.nelyan.ui

import android.content.Context
import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.ImageView
import android.widget.LinearLayout
import androidx.appcompat.app.AppCompatActivity
import com.nelyan.R
import com.nelyan.ui.ActivityFormActivity

class PubilerActivity : AppCompatActivity() {
    var mContext: Context? = null
    var ivBack: ImageView? = null
    var ivLogo: ImageView? = null
    var ivRadio1: ImageView? = null
    var ivOn: ImageView? = null
    var ivRadio2: ImageView? = null
    var ivRadio3: ImageView? = null
    var ivRadio4: ImageView? = null
    var ll_1: LinearLayout? = null
    var ll_2: LinearLayout? = null
    var ll_3: LinearLayout? = null
    var ll_4: LinearLayout? = null
    var ll_5: LinearLayout? = null
    var btnSubmit: Button? = null
    var state = 0
    var type = "1"
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_pubiler)
        mContext = this
        ivBack = findViewById(R.id.ivBack)
        ll_1 = findViewById(R.id.ll_1)
        ll_2 = findViewById(R.id.ll_2)
        ll_3 = findViewById(R.id.ll_3)
        ll_4 = findViewById(R.id.ll_4)
        ll_5 = findViewById(R.id.ll_5)
        ivOn = findViewById(R.id.ivOn)
        ivRadio1 = findViewById(R.id.ivRadio1)
        ivRadio2 = findViewById(R.id.ivRadio2)
        ivRadio3 = findViewById(R.id.ivRadio3)
        ivRadio4 = findViewById(R.id.ivRadio4)
        ll_1!!.setOnClickListener(View.OnClickListener {
            type = "1"
            ivOn!!.setImageResource(R.drawable.radio_outline)
            ivRadio1!!.setImageResource(R.drawable.radio_fill)
            ivRadio2!!.setImageResource(R.drawable.radio_outline)
            ivRadio3!!.setImageResource(R.drawable.radio_outline)
            ivRadio4!!.setImageResource(R.drawable.radio_outline)
        })
        ll_2!!.setOnClickListener(View.OnClickListener {
            type = "2"
            ivOn!!.setImageResource(R.drawable.radio_fill)
            ivRadio1!!.setImageResource(R.drawable.radio_outline)
            ivRadio2!!.setImageResource(R.drawable.radio_outline)
            ivRadio3!!.setImageResource(R.drawable.radio_outline)
            ivRadio4!!.setImageResource(R.drawable.radio_outline)
        })
        ll_3!!.setOnClickListener(View.OnClickListener {
            type = "3"
            ivOn!!.setImageResource(R.drawable.radio_outline)
            ivRadio1!!.setImageResource(R.drawable.radio_outline)
            ivRadio2!!.setImageResource(R.drawable.radio_fill)
            ivRadio3!!.setImageResource(R.drawable.radio_outline)
            ivRadio4!!.setImageResource(R.drawable.radio_outline)
        })
        ll_4!!.setOnClickListener(View.OnClickListener {
            type = "4"
            ivOn!!.setImageResource(R.drawable.radio_outline)
            ivRadio1!!.setImageResource(R.drawable.radio_outline)
            ivRadio2!!.setImageResource(R.drawable.radio_outline)
            ivRadio3!!.setImageResource(R.drawable.radio_fill)
            ivRadio4!!.setImageResource(R.drawable.radio_outline)
        })
        ll_5!!.setOnClickListener(View.OnClickListener {
            type = "5"
            ivOn!!.setImageResource(R.drawable.radio_outline)
            ivRadio1!!.setImageResource(R.drawable.radio_outline)
            ivRadio2!!.setImageResource(R.drawable.radio_outline)
            ivRadio3!!.setImageResource(R.drawable.radio_outline)
            ivRadio4!!.setImageResource(R.drawable.radio_fill)
        })
        ivBack!!.setOnClickListener(View.OnClickListener { finish() })
        ivLogo = findViewById(R.id.ivLogo)
        ivLogo!!.setOnClickListener(View.OnClickListener {
            val i = Intent(this@PubilerActivity, HomeActivity::class.java)
            startActivity(i)
        })
        btnSubmit = findViewById(R.id.btnSubmit)
        btnSubmit!!.setOnClickListener(View.OnClickListener {
            if (type == "1") {
                val i = Intent(this@PubilerActivity, ActivityFormActivity::class.java)
                startActivity(i)
            } else if (type == "2") {
                val i = Intent(this@PubilerActivity, NurserieActivity::class.java)
                startActivity(i)
            } else if (type == "3") {
                val i = Intent(this@PubilerActivity, MaternalAssistantActivity::class.java)
                startActivity(i)
            } else if (type == "4") {
                val i = Intent(this@PubilerActivity, BabySitterActivity::class.java)
                startActivity(i)
            } else if (type == "5") {
                val i = Intent(this@PubilerActivity, com.nelyan.ui.TraderActivity::class.java)
                startActivity(i)
            }
        })
    }
}